### ISSUE TITLE
Run dm-vio real-time 

### ISSUE THREAD

**Issue by trand2k** [lukasvst/dm-vio-python-tools#5]
Hi author, 
Can i read data IMU and camera from ros topic and run dm-vio realtime? 
thank you

**Comment by lukasvst** [comment]
Yes, there is a ROS version at https://github.com/lukasvst/dm-vio-ros
